twig/extensions/1.0/config/packages/twig_extensions.yaml

```diff
@@ -1,11 +0,0 @@
-services:
-    _defaults:
-        public: false
-        autowire: true
-        autoconfigure: true
-
-    # Uncomment any lines below to activate that Twig extension
-    #Twig\Extensions\ArrayExtension: null
-    #Twig\Extensions\DateExtension: null
-    #Twig\Extensions\IntlExtension: null
-    #Twig\Extensions\TextExtension: null
```

Comment:
This is also a case which currently is hard to handle. Examples which are indented somewhere in the tree.